cli/enry/main.go

```diff
@@ -19,22 +19,24 @@ var (
 )
 
 func main() {
-	flag.Usage = usage
-	breakdownFlag := flag.Bool("breakdown", false, "")
-	jsonFlag := flag.Bool("json", false, "")
-	flag.Parse()
+	args, flags := splitArgs()
```

Comment:
if an unknown flag (--help, for example) is found, it's appended to the flags map because has the prefix "-". 

Then, when you call parseFlags, the unknown flag is ignored but a usage message should be shown.